.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    branches:
+    - main
```

Comment:
Might be worth adding `master` here since I can see Butter uses the old default name of `master`